en/flight_modes_rover/manual.md

```diff
@@ -0,0 +1,88 @@
+# Manual Modes (Rover)
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/rover_manual_controls.png)
+
+The manual modes listed below provide increasing levels of autopilot support:
+
+| Mode                           | Features                                                                                                   |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------- |
+| [Manual](#manual-mode)         | Directly map stick inputs to motor commands, no closed loop control.                                       |
+| [Acro](#acro-mode)             | Closed loop yaw rate control.                                                                              |
+| [Stabilized](#stabilized-mode) | Closed loop yaw rate and yaw control.                                                                      |
```

Comment:
Ditto - the "both" might be

```suggestion
| [Stabilized](#stabilized-mode) | Closed loop yaw rate and yaw control (holds direction better when yaw is not applied).                                                                     |
```

Reply:
Added [here](https://github.com/PX4/PX4-user_guide/pull/3377/files#diff-dd65fbcd095c40befe925ee616cdb0679de431aac441983568baf7b8e23ed9c5R25),